control_flows_and_logical_operator/task_02.py

```diff
@@ -0,0 +1,11 @@
+password = str(input("Enter password: "))
+
+match password:
+    case p if len(p) < 8:
+        print("Password is invalid. Reason: At least 8 characters long.")
+    case p if " " in p:
+        print("Password is invalid. Contains spaces.")
+    case p if "Python" not in p :
+        print("Password is invalid. Does not contain the word 'Python'.")
+    case _:
+        print("Password is valid.")
```

Comment:
`match case` construction is redundant here because it does not use pattern-matching
It would be more simple and concise using regular `if` construction

```suggestion
if len(password) < 8:
    print("Password is invalid. Reason: At least 8 characters long.")
elif " " in password:
    print("Password is invalid. Contains spaces.")
elif "Python" not in password:
    print("Password is invalid. Does not contain the word 'Python'.")
else:
    print("Password is valid.")
```

Reply:
Replaced the match case with an if statement